vizro-ai/mkdocs.yml

```
@@ -1,5 +1,5 @@
 site_name: Vizro-AI
-site_url: https://mckinsey.github.io/vizro-ai/
+site_url: https://vizro.readthedocs.io/en/stable/
```

This might be very confusing, but vizro-ai doesn't have a stable version but only a latest. Best to double-check with @maxschulz-COL if we actually want to change that:

```suggestion
site_url: https://vizro-ai.readthedocs.io/en/latest/
```

---

Yes, this is important. Thanks for pointing this out @huong-li-nguyen!

Vizro-ai does not have stable docs, but only latest, as RTD does not (easily? didn't check in a wile) allow multiple sets of docs in a monorepo that also link to that repos release cycle.

Let's discuss more in person, as this also somewhat links to the discussion if we should have two sets of docs.

---

Ah yes, good point about the latest vs stable. I feel like all the signs are pointing to building docs together in one or setting up a proper subproject in RTD so that the `vizro-ai` docs can have its own release cadence and then latest + stable docs. I am not sure how easily this can be done with MkDocs compared to Sphinx but will investigate. I would, however, very much like to look at whether it's worth having separate docs at all, so will take a more holistic view and compare both approaches before trying to solve for just one.

@maxschulz-COL The `site_url` setting is to set the canonical URL, which in SEO terms means that, where identical content appears in more than one place (e.g. repo in markdown and published as HTML on RTD) it tells search engines which to surface in preference. So we do actually want the `site_url` to be the docs rather than the repo. This change is the correct thing to do for that, and has the benefit of fixing the 404 page alongside!